contractcourt/htlc_success_resolver.go

```diff
@@ -200,7 +199,7 @@ func (h *htlcSuccessResolver) Resolve() (ContractResolver, error) {
 	//
 	// TODO(roasbeef): after changing sighashes send to tx bundler
 	err := h.PublishTx(h.htlcResolution.SignedSuccessTx)
-	if err != nil && err != lnwallet.ErrDoubleSpend {
+	if err != nil {
```

Comment:
So with this change, do we even still need `lnwallet.ErrDoubleSpend` at all?

Reply:
The error is still needed with its current purpose, as pointed out by @joostjager. Its former purpose of detecting whether the transaction is in the mempool or has already confirmed is no longer needed as we won't return an error in those cases.